apps/javascript/src/challenges/color-me/index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>3x3 Round Grid Color Me</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script src="../../helpers/header.js" type="module"></script>
+  </head>
+  <body>
+    <div class="container">
+      <h1>3x3 Grid Color Me</h1>
+
+      <div class="grid-container" id="grid"></div>
+
+      <div class="button-container">
+        <input type="number" id="inputBox" min="1" max="9" placeholder="Enter" />
+        <button id="colorButton">Color Me</button>
+      </div>
```

Comment:
This should be a form. The value should be taken when the **Enter** key is pressed